homeassistant/components/climate/fritzhome.py

```diff
@@ -0,0 +1,138 @@
+"""
+Support for AVM Fritz!Box fritzhome thermostate devices.
+
+For more details about this component, please refer to the documentation at
+http://home-assistant.io/components/climate.fritzhome/
+"""
+import logging
+
+from components.fritzhome import (ATTR_AIN, ATTR_FW_VERSION,
+    ATTR_ID, ATTR_MANUFACTURER, ATTR_PRODUCTNAME, DOMAIN)
+from homeassistant.components.climate import (ClimateDevice, STATE_ECO)
+from homeassistant.const import (PRECISION_HALVES)
+from homeassistant.const import (TEMP_CELSIUS, ATTR_TEMPERATURE)
+
+DEPENDENCIES = ['fritzhome']
+
+_LOGGER = logging.getLogger(__name__)
+
+STATE_COMFORT = 'comfort'
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the Fritzhome thermostat platform."""
+
+    if DOMAIN not in hass.data:
+        return False
+
+    device_list = hass.data[DOMAIN]
+
+    devices = []
+    for device in device_list:
+        if device.has_thermostat:
+            devices.append(FritzhomeThermostat(hass, device))
+
+    add_devices(devices)
+
+
+class FritzhomeThermostat(ClimateDevice):
+    """The thermostat class for Fritzhome."""
+
+    OPERATION_LIST = [STATE_ECO, STATE_COMFORT]
```

Comment:
Why do you define this as a member? I would define this with the other constants, where you defined `STATE_COMFORT`